tools/cythonize.py

```diff
@@ -100,6 +100,24 @@ def process_tempita_pyx(fromfile, tofile):
         f.write(pyxcontent)
     process_pyx(pyxfile, tofile)
 
+
+def process_tempita_pxi(fromfile, tofile):
+    try:
+        try:
+            from Cython import Tempita as tempita
```

Comment:
Robert Bradshaw, the primary Cython developer, says "No, it's just an implementation detail," in response to someone asking if the vendorized version of Tempita in the Cython sources was meant to be used. If he says that we should not rely on it, then we should not rely on it. He can remove it or modify it on a whim.

https://groups.google.com/d/msg/cython-users/LViuQUs4iQo/-fDcdNUFBwAJ